canvas/image.go

```diff
@@ -38,8 +48,9 @@ type Image struct {
 	Resource fyne.Resource // Load the image from an in-memory resource
 	Image    image.Image   // Specify a loaded image to use in this canvas object
 
-	Translucency float64   // Set a translucency value > 0.0 to fade the image
-	FillMode     ImageFill // Specify how the image should scale to fill or fit
+	Translucency float64    // Set a translucency value > 0.0 to fade the image
+	FillMode     ImageFill  // Specify how the image should expand to fill or fit the available space
+	ScaleMode    ImageScale // Specify how type of scaling applied to the image
```

Comment:
This sentence doesn’t make sense - did you mean “Specify the type” instead of “Specify how type”?

Reply:
I just merged @stuartmscott change ;-)

Reply:
I don’t think that what exists in the result is what either of us suggested

Reply:
There is a typo in the change I suggested, but let's not merge it into the repo

Reply:
Oh yes I see it now. Probably best to sanity check suggestions ;)